container/new_push.bzl

```diff
@@ -165,7 +183,7 @@ new_container_push = rule(
             doc = "(optional) The label of the file with tag value. Overrides 'tag'.",
         ),
         "_digester": attr.label(
-            default = "@containerregistry//:digester",
+            default = Label("@io_bazel_rules_docker//container/go/cmd/digester:digester"),
```

Comment:
nit: the `Label` is not needed. Also, just `"@io_bazel_rules_docker//container/go/cmd/digester"` should work for the label.